pkg/store/kotsstore/preflight_store_test.go

```diff
@@ -126,7 +126,7 @@ func Test_hasFailingStrictPreflights(t *testing.T) {
 			name:               "expect false, error when preflightSpec with strict:true analyzer and preflightResultSpec has a empty string",
 			preflightSpecStr:   toSqlString(t, strictTruePreflightSpec),
 			preflightResultStr: gorqlite.NullString{Valid: true, String: ""},
-			want:               true,
+			want:               false,
```

Comment:
This seems like it was an error in the tests. The `name` of this test indicates that we would expect `false` when the preflight results are empty. This makes sense since how could we determine if the version  `hasFailingStrictPreflights` if we have no results to check?